docs/workbench/notebook/jupyter.yaml

```diff
@@ -0,0 +1,60 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements. See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License. You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+kind: Deployment
+apiVersion: extensions/v1beta1
+metadata:
+  name: jupyter-deployment
+  labels:
+    app: jupyter
+spec:
+  selector:
+    matchLabels:
+      app: jupyter
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        namespace: submarine
+        app: jupyter
+    spec:
+      nodeSelector:
+        kubernetes.io/hostname: k8s-submarine-control-plane
+      containers:
+        - name: jupyter
+          image: jupyter/minimal-notebook
+          # mount path in jupyter container
+          volumeMounts:
+            - mountPath: /home/jovyan
```

Comment:
No personal information can appear in the project code and documentation, so please modify the following path `/home/jovyan` change to such as `/tmp/submarine` or other?